server/src/main/java/org/elasticsearch/index/mapper/DateFieldMapper.java

```diff
@@ -913,6 +914,17 @@ protected String contentType() {
 
     @Override
     protected void parseCreateField(DocumentParserContext context) throws IOException {
+        if (context.parser().currentToken() == XContentParser.Token.START_OBJECT) {
+            if (ignoreMalformed) {
+                context.addIgnoredField(mappedFieldType.name());
+                if (isSourceSynthetic) {
+                    context.doc().add(IgnoreMalformedStoredValues.storedField(name(), context.parser()));
+                }
+                return;
+            } else {
+                throw new IllegalArgumentException("Unable to parse object as a " + mappedFieldType.name() + " field");
```

Comment:
Paranoid check - isn't this an API change? Previously we would have returned an error directly from parse which is not only a different text but a different type of exception.

Reply:
I will double check this...probably I will add also `full-bwc` label.

Reply:
AT least I hope this i test...but probably it is not...

Reply:
Yea i don't know if we explicitly test such things.